frodoProtocols.m

```diff
@@ -406,6 +417,23 @@
 end
 
 
+%%
+function [pulseCommandArray] = currentStepPulse( pulseAmp, pulseDur )
+ephysSettings;
+
+BREAK_DURATION = 0.05; %sec
+
+preStepCommand = zeros(1, BREAK_DURATION * rigSettings.sampRate );
+
+stepCommand = pulseAmp * ones( 1, pulseDur * rigSettings.sampRate );
+
+pulseCommand = [preStepCommand stepCommand preStepCommand];
+
+pulseCommandArray = pulseCommand * rigSettings.command.currentClampExternalCommandGain; % send full command out, in Voltage for the daq to send
```

Comment:
Looks prefect!   Unrelated to this commit, but is this gain value from the 700B now saved from the telegraph so we can confirm later if this was ever wrong?